sdk/servicebus/service-bus/test/atomXml.spec.ts

```diff
@@ -72,6 +76,59 @@ describe("atomSerializationPolicy #RunInBrowser", function() {
   });
 });
 
+describe("deserializeAtomXmlResponse #RunInBrowser", function() {
+  it("should throw an error if receiving a valid XML but invalid Atom XML", async function() {
+    try {
+      await serviceBusAtomManagementClient.deleteQueue("alwaysBeExistingQueue");
```

Comment:
Actually, for this test you probably don't even need to make a real request. You can just create a `_response` object that has the fields you need set (status, parsedBody, request if needed).

Reply:
That's what I had first tried but kept getting a 'cannot read property clone() on undefined' error.
Still not sure where that is coming from..

Reply:
Looked into this a little more, looks like it needed the headers/request to be constructed properly - updated to use mocked functionality